src/Symfony/Bridge/Doctrine/SchemaListener/AbstractSchemaListener.php

```diff
@@ -11,27 +11,11 @@
 
 namespace Symfony\Bridge\Doctrine\SchemaListener;
 
-use Doctrine\Common\EventSubscriber;
 use Doctrine\DBAL\Connection;
 use Doctrine\DBAL\Exception\TableNotFoundException;
-use Doctrine\ORM\Tools\Event\GenerateSchemaEventArgs;
-use Doctrine\ORM\Tools\ToolEvents;
 
-abstract class AbstractSchemaSubscriber implements EventSubscriber
+abstract class AbstractSchemaListener
```

Comment:
Userland code might have extended `AbstractSchemaSubscriber` directly. I think we need to keep the class for BC and deprecate it instead.

Reply:
This class is new to 6.3 ;)